ts/features/itwallet/lifecycle/store/actions/index.ts

```diff
@@ -0,0 +1,26 @@
+import { ActionType, createStandardAction } from "typesafe-actions";
+
+export const itwLifecycleInstalled = createStandardAction(
+  "ITW_LIFECYCLE_INSTALLED"
+)<void>();
+
+export const itwLifecycleOperational = createStandardAction(
+  "ITW_LIFECYCLE_OPERATIONAL"
+)<void>();
+
+export const itwLifecycleValid = createStandardAction(
+  "ITW_LIFECYCLE_VALID"
+)<void>();
+
+export const itwLifecycleDeactivated = createStandardAction(
+  "ITW_LIFECYCLE_DEACTIVATED"
+)<void>();
```

Comment:
What do you think about one single parametrized action? It could be easier to use